### DISCUSSION TITLE
Housing data

### DISCUSSION THREAD

**Discussion by jacobrharris** [datakind/datakit-housing-fall-2024#13] [Challenges]
<hr /><em>This discussion was created from the release <a href='https://github.com/datakind/datakit-housing-fall-2024/releases/tag/v1.0'>Housing data</a>.</em>

**Comment by KathrynMercer** [comment]
I looked through this data and made a high level (human eye friendly) description of what it contains. Expand below or download at your preference. Feel free to update this file as we go. You can also browse this info on [DataKind's Economic Opportunity Datascape](https://eodatascape.datakind.org/explore-data), but I like having it here.

For most of the current challenges, I expect data_1 will be most helpful. 

[Data Explanation.txt](https://github.com/user-attachments/files/17041559/Source.Explanation.txt)

### data_1-CA.csv + data_1-FL.csv
<details>
  <summary>Geographically organized population data (both states have the same fields)</summary>

  - **Fields with null data**:
    - `loan_amount`
    - `median_mortgage_amount`
    - `median_prop_value`
    - `median_sba504_loan_amount`
    - `median_sba7a_loan_amount`
    - `num_mortgage`
    - `num_mortgage_denials`
    - `num_mortgage_originated`
    - `number_of_sba504_loans`
    - `number_of_sba7a_loans`
  
  - **Fields with untrustworthy data**: some large value negatives (not yet investigated thoroughly)
  
  - **Redundant fields**:
    - `geoid_year`
    - `state`
    - `state_fips_code`
    - `county_fips_code`

  #### Field Description Summary:

  - **Signals of Economic Difficulty**:
    - `economic_distress_pop_agg`: 20+% of households in the census tract are very low-income renters or owners who pay more than half their income for housing, population weighted
    - `economic_distress_simple_agg`: 20+% of households in the census tract are very low-income renters or owners who pay more than half their income for housing, NOT population weighted
    - `investment_areas`: geographic area with:
      1. Population poverty rate of 20%+
      2. Unemployment rate of at least 1.5 times the national average
      3. Metropolitan areas with median family income (MFI) at or below 80% of either the metropolitan or national MFI
      4. Non-metropolitan areas with MFI at or below 80% of either the statewide or national non-metropolitan MFI
      5. Wholly located within an empowerment zone or enterprise community
      6. County population loss greater than or equal to 10% for metro areas or 5% for non-metro areas
    - `num_mortgage`: number of home mortgage loans reported
    - `num_mortgage_denials`: number of home mortgage loans denied
    - `num_mortgage_originated`: number of home loans originated
    - `opzone`: Opportunity Zones are census tracts identified in areas of economic distress to promote investment (boolean 1/0)
    - `b19083_001e`: Gini Index of Income Inequality - Estimate

  - **About the Homes**:
    - `median_mortgage_amount`: median home mortgage amount
    - `median_prop_value`: median home property value amount

  - **About Demographics**:
    - "B Series" (e.g., `b23025_002e`): employment percentages (9 rows)
    - "s0101 Series" (e.g., `s0101_c01_032e`): census data organizing population by age, race, sex, etc. (78 rows)
    - "dp05 Series" (e.g., `dp05_0035pe`): census data organized by race (7 rows)
    - "s2001 Series" (e.g., `s2001_c01_002e`): median earnings total vs split by sex (6 rows)
    - "s1903 Series" (e.g., `s1903_c03_001e`): median income total vs split by race (28 rows)
    - "s1701 Series" (e.g., `s1701_c03_001e`): percentage of people below poverty level total vs split by age, race (40 rows)
    - "s2701 Series" (e.g., `s2701_c03_001e`): health insurance coverage split by race, sex, etc. (94 rows)

  - **Other**:
    - SBA loan information (e.g., `number_of_sba504_loans`): small business loan issued (4 rows)
    - `loan_amount`: original loan/investment amount (unclear if this is for home loans)

  #### Data Dictionary:
  - Contains descriptions, sources, and years for information in data_1.

</details>

---

### data_2-CA.csv (California only)
<details>
  <summary>Geographically organized information about health and natural hazard risks (mostly from EPA)</summary>

  #### Housing/Demographics:

  - "B Series": employment percentages (4 rows), e.g., `b23025_003e`
  - "S2503 Series": occupied housing organized by total, rented, owned homes (6 rows), e.g., `s2503_c01_024e`
  - Energy Burden: percentage of gross household income spent on energy costs (2 rows), e.g., `energy_burden`

  #### Hazard Data:

- **Air Quality**:
  - Air Cancer: air toxic cancer risk info (6 rows), e.g., `p_d2_cancer`
  - Respiratory Tract Toxins: air toxics respiratory risk info (6 rows), e.g., `d2_resp`
  - Air Toxins: toxic releases to air (6 rows), e.g., `d2_rsei_air`
  - Ozone: ozone risk info (6 rows), e.g., `d2_ozone`
  - Particulate Matter: particulate matter risk info (6 rows), e.g., `d2_pm25`
  - Diesel Particulate Matter: diesel particulate matter risk info (6 rows), e.g., `d2_dslpm`

- **Lead Paint**:
  - EPA Lead Paint Risk: lead paint exposure risk as estimated by the EPA (5 rows), e.g., `d2_ldpnt`
  - EJScreen Lead Paint Indicator: lead paint exposure based on housing age from EJScreen (2 rows), e.g., `pre1960` 

- **Proximity to Hazardous Sites**:
   - Superfund Proximity: measures proximity to Superfund sites (6 rows), e.g., `d2_pnpl`
   -- Superfund programs: 2014 program to identify, clean up, and return contaminated sites to productive use)
   - RMP Facility Proximity: proximity to Risk Management Program facilities (6 rows), e.g., `d2_prmp`
   -- RMP facility: facilities that require a Risk Management Program because they use hazardous substances
   - Hazardous Waste Proximity: hazardous waste proximity (6 rows), e.g., `d2_ptsdf`
   - Traffic Proximity: proximity to high traffic areas (6 rows), e.g., `d2_ptraf`
   - Wastewater Discharge: wastewater discharge risk info (6 rows), e.g., `d2_pwdis`
   - Underground Storage Tanks: underground storage tank exposure (6 rows), e.g., `d2_ust`

 #### Expected Losses due to Natural Hazards:

  - Agricultural Loss: expected agricultural loss rate (2 rows), e.g., `expected_agricultural_loss_rate_natural_hazards_risk_index`
  - Building Loss: expected building loss rate (2 rows), e.g., `expected_building_loss_rate_natural_hazards_risk_index`
  - Population Loss: expected population loss rate (2 rows), e.g., `expected_population_loss_rate_natural_hazards_risk_index`
  - Fire Risk: properties at risk of fire within 30 years (2 rows), e.g., `share_of_properties_at_risk_of_fire_in_30_years`
  - Flood Risk: properties at risk of flood within 30 years (2 rows), e.g., `share_of_properties_at_risk_of_flood_in_30_years`

  #### Other:

  - `QCT`: Qualified Census Tract Indicator, 1 row, `qct` (just zeros)
  - `pre1960`: associated with lead paint exposure, but could also be used to look at housing age

  #### Data Dictionary:
  - Contains descriptions, sources, and years for information in data_2.

</details>

**Comment by ChenetDK** [comment]
For those interested in diving into different data sets, the Administration for Community Living (ACL) and the Department of Housing and Urban Development (HUD) have also collected relevant data sets through their Housing and Services Partnership Accelerator program.

 You can find more information on their website: [link](https://acl.gov/HousingAndServices/data-and-reports).

**Comment by MELSHAMA** [comment]
I renamed columns in data_1-FL.csv guided by the desc in data dictionary for more readability.Here is the result dataset
[data_1-FL_renamed.csv](https://github.com/user-attachments/files/17087818/data_1-FL_renamed.csv)
